site/owid.scss

```diff
@@ -631,11 +631,12 @@ div.owid-presentations li:nth-of-type(1) a:hover {
 .with-sidebar .section-heading {
     @include xxlg-up {
         position: relative;
-        width: calc(100vw - #{$sidebar-content-width + 2 * $padding-x-md});
-        margin-left: calc(
-            50% - 50vw - #{($sidebar-content-width + 2 * $padding-x-md) * 0.5}
+        width: calc(100vw - $sidebar-width);
+        margin-left: 0;
+        left: min(
+            calc((100vw - 100% - $sidebar-width) / -2),
+            calc(-1 * $padding-x-md)
```

Comment:
This is pretty heinous but here's what's going on.

![magic margin](https://user-images.githubusercontent.com/11844404/191846428-fd6470f9-0baf-4afa-93b7-0ac3a27b7606.jpg)

`100vw - 100% - $sidebar-width` gives us a width equivalent to the two `remainder` rectangles in the illustration. 

We divide that by two so that we just have the remainder on the left, and make it negative.

The `min` function is needed because there is a small window where this formula doesn't work. 🙂